net_conn.go

```diff
@@ -15,11 +12,15 @@ type netConn struct {
 
 // writeString formats syslog messages using time.RFC3339 and includes the
 // hostname, and sends the message to the connection.
-func (n *netConn) writeString(p Priority, hostname, tag, msg string) error {
-	timestamp := time.Now().Format(time.RFC3339)
-	_, err := fmt.Fprintf(n.conn, "<%d>%s %s %s[%d]: %s",
-		p, timestamp, hostname,
-		tag, os.Getpid(), msg)
+func (n *netConn) writeString(framer Framer, formatter Formatter, p Priority, hostname, tag, msg string) error {
+	if framer == nil {
```

Comment:
To simplify the initialization of the `srslog.Writer` (and anything else that might use a `netConn`, not that any such thing currently exists).

I wanted to localize the default formatter and framer closer to where they're going to be used, rather than have to know in a different place that you have to set up defaults in order to call this function.